data-model/src/main/scala/za/co/absa/enceladus/model/menas/audit/Auditable.scala

```diff
@@ -79,6 +79,74 @@ trait Auditable[T <: Product] { self: T =>
     }).getOrElse(Seq())
   }
 
+
+
+  private[model] def getOptionalMapFieldsAudit(newValue: T, fieldName: AuditFieldName)(implicit ct: ClassTag[T]): Seq[AuditTrailChange] = {
+    def tupleToSomeString(v: (String, _)): Option[String] = Some(s"""key: "${v._1}" value: "${v._2}"""")
```

Comment:
**Minor:** The function name is not very informative IMHO. What about `makeAuditTrailChangeValue`? Or anything 😄

Reply:
Followed convention of `getPrimitiveFieldsAudit`, `getSeqFieldsAudit`. Would you change these as well?

Reply:
Sorry not being clear. No problem with `getOptionalMapFieldsAudit`.
I was talking about `tupleToSomeString`.